### ISSUE TITLE
Local channels [DENMARK]

### ISSUE THREAD

**Issue by jnylen** [xmltv-se/xmltv.se#5]
I'm working on adding the local channels in Denmark. It's hard to get in contact with them though.

- [ ] 24Nordjyske
- [ ] TV Syd
- [ ] TV2 Fyn
- [ ] TV2 Nord Salto
- [ ] TV2 Østjylland
- [ ] Lokalkanalen
- [ ] Hovedstaden
- [x] TV Bornholm
- [x] TV Midtvest

**Comment by twister5800** [comment]
Hi, 
Why aren't the xmltv.xmltv.se/channels-Denmark.xml.gz not being updates anymore?

Br. martin

**Comment by jnylen** [comment]
The file only gets updated when an channel gets edited, removed or added.

**Comment by supermax45** [comment]
VH1 Classic DK is missing
Discovery HD Showcase with DK epg is missing

Both was available before the server breakdown.

I know it's not a danish channel, but Trace Urban would be nice to have epg for :)

**Comment by jnylen** [comment]
D HD Showcase (DK) -> hdshowcase.discovery.dk
D HD (DK) -> hd.discovery.dk

TV2 OJ and Esports tv is in

Vh1 Classic: I need to find the ID they had for the data. Probably later today
Radiostations is being looked at aswell.

**Comment by supermax45** [comment]
None of them is to find in the combined xml file? Will they also be added in the combined or do i have to combine it by myself?

**Comment by supermax45** [comment]
??

**Comment by jnylen** [comment]
Combined files isn't supported. You can make a pull request for the configs files at xmltv-se/configs and i'll update them.

**Comment by jecdk** [comment]
Will EPG data for Danish radio channels return?
The channels (P1, P2 etc.) are present, but data has been missing for several month.

**Comment by jnylen** [comment]
I haven't made a decision about that yet.

**Comment by jecdk** [comment]
Yes, understand that there probably are more important issues to spend time on, and thanks for doing that work. Just that I occasionally record form radio, which is so much easier when EPG is available.

**Comment by jnylen** [comment]
Hi @jecdk,
I added the DR channels back.
But we lost some of the old radiostations as we moved to the same API as the DR tv channels.
Which are the Radio ABC etc.

**Comment by jecdk** [comment]
Hi @jnylen,
Thanks for bringing back the DR radio. By now I however still find no data, f.ex.;
http://xmltv.xmltv.se/p1.dr.dk_2016-09-04.xml
is empty.

**Comment by jnylen** [comment]
Data is updated daily. Not instant. So tomorrow around 7 it should be there.

**Comment by supermax45** [comment]
Will you add epg for C More Stars just released 1 september? :)

**Comment by jnylen** [comment]
@supermax45: It's there already. action.cmore.dk is Stars, emotion.cmore.dk is Stars HD.

**Comment by supermax45** [comment]
Okay thanks :+1:

**Comment by jecdk** [comment]
Hi @jnylen,
Thanks, DR radio EPG is now working.
